elasticsearch_collectd.py

```diff
@@ -722,10 +722,19 @@ def __str__(self):
                 attrs.append("{}={}".format(name, getattr(self, name)))
         return "<CollectdValues {}>".format(' '.join(attrs))
 
+def set_es_cluster_from_host():
+        global ES_CLUSTER
+        json = fetch_url("http://" + ES_HOST + ":" + str(ES_PORT))
```

Comment:
this is already loaded in load_es_version() below. Can you move the logic there.
you also don't need line 731 since the variable is already initialized to that value.